tutorials/quick_start/pytorch_fw/timm/model_lib_timm.py

```diff
@@ -0,0 +1,58 @@
+import timm
+from timm.data import create_dataset, create_loader, resolve_data_config
+
+from common.model_lib import BaseModelLib
+from pytorch_fw.utils import classification_eval, get_representative_dataset
+from common.consts import MODEL_NAME, BATCH_SIZE, VALIDATION_SET_LIMIT, VALIDATION_DATASET_FOLDER
+
+from tutorials.quick_start.common.results import DatasetInfo
+
+
+class ModelLib(BaseModelLib):
+
+    def __init__(self, args):
+        avialable_models = timm.list_models('')
+        if args[MODEL_NAME] in avialable_models:
+            self.model = timm.create_model(args[MODEL_NAME], pretrained=True)
+            self.data_config = resolve_data_config([], model=self.model)  # include the pre-processing
+            self.dataset_name = 'ImageNet'
```

Comment:
Add a comment stating that this value is constant because we currently support only timm models that are trained on Imagenet (or something like that)